androidHyperskillApp/src/main/res/layout/layout_problems_limit.xml

```diff
@@ -23,28 +30,32 @@
             android:layout_height="wrap_content"
             android:textColor="@color/color_on_surface"
             android:layout_marginEnd="8dp"
-            tools:text="3/5 problems limit" />
+            android:layout_gravity="center_vertical"
+            tools:text="3/5 problems left" />
 
         <TextView
             android:id="@+id/problemsLimitUpdatedIn"
             android:layout_width="wrap_content"
-            android:layout_height="match_parent"
+            android:layout_height="wrap_content"
             android:textColor="@color/color_on_surface_alpha_38"
             android:textSize="12sp"
             android:gravity="bottom"
-            tools:text="Update in 2 h."/>
+            android:layout_gravity="center_vertical"
+            tools:text="Reset in 2 h. Reset in 2 h. Reset in 2 h. Reset in 2 h."/>
```

Comment:
it seems better to leave "Reset in 2 h." Like we have it in design

Reply:
@vladkash 
A long string is used here to test the behavior in the case of a small device when this text view doesn't have enough width.